src/test/java/PasswordGeneratorTest.java

```diff
@@ -49,10 +48,12 @@ public void checkPasswordLength() {
 
     @Test
     public void checkSecretPassword() {
+        //do we need this variable?
         BCryptPasswordEncoder bCryptPasswordEncoder = new BCryptPasswordEncoder();
 
         String actualPassword;
 
+        //valid but can we do better?
         while (true) {
```

Comment:
Whilst this loop works fine some people might object to the style of having an infinite loop with a break statement.  Can you think of a way to re-write it?